SYCL/Matrix/joint_matrix_bfloat16_rowmajorA_rowmajorB.cpp

```diff
@@ -0,0 +1,29 @@
+//==--joint_matrix_bfloat16_rowmajorA_rowmajorB.cpp  - DPC++ joint_matrix---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
+
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// CHECK: passed
```

Comment:
This only has an effect if you run the output through FileCheck. It would be easier to just have the main functions return a non-0 value if they fail.

Reply:
maybe later we change all of it. other tesctcases have such issue as well.

Reply:
If this is indeed a problem for the other tests, I am okay with merging it as-is, but it should be addressed ASAP. Note that anything that can cause "failed" to be printed will not currently be considered a failure by the test system for these tests.

Reply:
Patch for fixing this: https://github.com/intel/llvm-test-suite/pull/1432